users/README.md

```diff
@@ -13,6 +13,7 @@
   'designation': string,
   'img': string,
   'github_id': string,
+  'github_user_id': string,
```

Comment:
GitHub api return I'd as an integer can we set the type of integer instead of string?

Reply:
Initially, I was doing the same, In one of my commits, I typecast the value because everything in the document is of type string, because it offers flexibility, is more readable,  compatibility, and conversions, strings can be easily converted to integers, but the opposite is not always true.